mpd/plugin.yaml

```diff
@@ -5,21 +5,47 @@ plugin:
     description:
         de: 'Music Player Deamon (MPD) Unterstützung'
         en: 'Music Player Deamon (MPD) support'
-    maintainer: '? (mknx)'
-#    tester: efgh                   # Who tests this plugin?
-#    keywords: kwd1 kwd2            # keywords, where applicable
-#    documentation: https://github.com/smarthomeNG/plugins/blob/develop/mpd/README.md # url of documentation (wiki) page
+    maintainer: 'E3EAT'
+#    tester:                        # Who tests this plugin?
+     keywords: MPD Music            # keywords, where applicable
+#    documentation: https://github.com/smarthomeNG/plugins/blob/develop/mpd/...md # url of documentation (wiki) page
 
 # Following entries are for Smart-Plugins:
-#    version: 1.3.3                 # Plugin version
-#    sh_minversion: 1.3             # minimum shNG version to use this plugin
+    version: 1.3.1                 # Plugin version
+    sh_minversion: 1.3             # minimum shNG version to use this plugin
 #    sh_maxversion:                 # maximum shNG version to use this plugin (leave empty if latest)
-#    multi_instance: False
+    multi_instance: True
     classname: MPD                 # class containing the plugin
 
 #parameters:
     # Definition of parameters to be configured in etc/plugin.yaml
-    
```

Comment:
I guess the following parameters are plugin parameters. So you need to uncomment the `parameters` section here.